src/main/java/interface_adapter/club_logged_in/create_post/CreatePostState.java

```diff
@@ -8,8 +11,8 @@
 public class CreatePostState extends ClubLoggedInState {
     private String title;
     private String content;
-    private String timeOfPosting;
```

Comment:
using LocalTime and LocalDate will allow us to compare better for organizing the events later. good idea.

Reply:
Thanks!